### PR TITLE
Don't mask `this` with nil

### PR DESCRIPTION
**Pull request by dmohs**
https://broadinstitute.atlassian.net/browse/GAWB-1478

- [x] **Submitter**: Include the JIRA issue number in the PR description
- [x] **Submitter**: Check documentation and code comments. Add explanatory PR comments if helpful.
- [x] **Submitter**: JIRA ticket checks:
  * Acceptance criteria exists and is met
  * Note any changes to implementation from the description
  * Add notes on what you've tested
- [x] **Submitter**: Update RC_XXX release ticket with any config or environment changes necessary
- [x] **Submitter**: Update FISMA documentation if changes to:
  * Authentication
  * Authorization
  * Encryption
  * Audit trails
- [x] **Submitter**: If you're adding new libraries, sign us up to security updates for them
- [x] Tell the tech lead (TL) that the PR exists if they wants to look at it
- [x] Anoint a lead reviewer (LR). **Assign PR to LR**
* Review cycle:
  * LR reviews
  * Rest of team may comment on PR at will
  * **LR assigns to submitter** for feedback fixes
  * Submitter rebases to develop again if necessary
  * Submitter makes further commits. DO NOT SQUASH
  * Submitter updates documentation as needed
  * Submitter **reassigns to LR** for further feedback
- [ ] **TL** sign off
- [x] **LR** sign off
- [x] **Product Owner** sign off
- [ ] **Assign to submitter** to finalize
- [x] **Submitter**: Verify all tests go green, including CI tests
- [ ] **Submitter**: Squash commits and merge to develop
- [ ] **Submitter**: Delete branch after merge
- [ ] **Submitter**: **Test this change works on dev environment after deployment**. YOU own getting it fixed if dev isn't working for ANY reason!
- [ ] **Submitter**: Mark JIRA issue as resolved once this checklist is completed

### CODE REVIEW

File: src/cljs/org/broadinstitute/firecloud_ui/endpoints.cljs [hunk 1]
```diff
@@ -710,7 +710,11 @@
    (call-ajax-orch
     {:endpoint {:path "/profile/billing"
                 :method :get
-                :mock-data (utils/rand-subset ["broad-dsde-dev" "broad-institute"])}
+                :mock-data (utils/rand-subset
```
Comment (zarsky-broad) [inline src/cljs/org/broadinstitute/firecloud_ui/endpoints.cljs:713]: I presume this is functionally unrelated to the problem, yes?

File: src/cljs/org/broadinstitute/firecloud_ui/page/billing/billing_management.cljs [hunk 2]
```diff
@@ -86,7 +86,7 @@
          (add-right
           [comps/Button
            {:text "Create New Billing Project"
-            :onClick (fn [{:keys [this]}]
+            :onClick (fn []
                        (if (-> @utils/google-auth2-instance (aget "currentUser") (js-invoke "get")
```
Comment (zarsky-broad) [inline src/cljs/org/broadinstitute/firecloud_ui/page/billing/billing_management.cljs:90]: could we not just simplify this to `:onClick #(if …`?

File: src/cljs/org/broadinstitute/firecloud_ui/endpoints.cljs (hunk 1 above)
Comment (dmohs) [reply src/cljs/org/broadinstitute/firecloud_ui/endpoints.cljs:713]: Correct, but it did help with development.

File: src/cljs/org/broadinstitute/firecloud_ui/page/billing/billing_management.cljs (hunk 2 above)
Comment (dmohs) [reply src/cljs/org/broadinstitute/firecloud_ui/page/billing/billing_management.cljs:90]: I prefer this form when the function is beyond one or two lines.

**Review by zarsky-broad — APPROVED** [review]

**Pull request merged**